zdict/dictionaries/oxford.py

```diff
@@ -0,0 +1,180 @@
+import json
+import os
+import re
+
+from zdict.constants import BASE_DIR
+from zdict.dictionary import DictBase
+from zdict.exceptions import NotFoundError, QueryError, ApiKeyError
+from zdict.models import Record
+
+
+KEY_FILE = 'oxford.key'
```

Comment:
Can we move this line into `_get_app_key()`?
Because when I read the docstring of `_get_app_key()`, I've already forgot what `KEYFILE` is and had to search it.
Is there any reason that we have to put this constant here?